<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutMeChefPart</class>
 <widget class="QWidget" name="AboutMeChefPart">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Cooked Dishes:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Time Difference 
Compared to Average</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>220</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Average Dish Rate</string>
   </property>
  </widget>
  <widget class="QLabel" name="dishNumLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>0</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeDifferenceLabel">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="rateInfo">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>195</y>
     <width>200</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
